cve_bin_tool/cve_scanner.py

```diff
@@ -204,28 +204,36 @@ def get_cves(self, product_info: ProductInfo, triage_data: TriageData):
 
             if cves:
                 self.products_with_cve += 1
-                self.logger.info(f"Known CVEs in {product_info}")
+
+                # TODO: code below should be enabled in verbose
+                # self.logger.info(f"Known CVEs in {product_info}")
+
```

Comment:
```suggestion
                self.logger.debug(f"Known CVEs in {product_info}")
```

I think I'd rather leave things available to people in debug mode *or* remove them from the code entirely if we don't think anyone will use them.  This one's pretty short so let's just leave it in.